### DISCUSSION TITLE
Library directories

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [premake/premake-core#1837] [Q&A] [answered]
How do I Define a library directory for the linker in Premake5.lua?
How do I also define dependencies in Premake5.lua?

(Library as in .lib files)

**Comment by KyrietS** [comment, marked as answer]
Have you seen [Linking](https://premake.github.io/docs/Linking/) page? There are a lot of examples in our documentation 🤠

**Reply by Ethan-Bit-256** [reply]
Thanks! I still find the wiki and site hard to use tbh.

**Reply by KyrietS** [reply]
Search box on our website is your friend. It works amazingly well. If you see any room for improvement, you can always let us know :-)

**Reply by Ethan-Bit-256** [reply]
Thanks, worked perfectly! sorry I forgot to respond btw
